src/clamav_report/clamav_report.py

```diff
@@ -213,23 +187,20 @@ def create_host_row(host_results):
     """Create a row of data from a host's results."""
     facts = host_results["Gathering Facts"][0]["ansible_facts"]
     # extract the mtimes from the "stat" module invocations
-    mtimes = dict()
+    mtimes = {}
     for stat_task in host_results["stat"]:
         path = stat_task["invocation"]["module_args"]["path"]
         mtime = stat_task["stat"].get("mtime", 0)  # 0 if it doesn't exist
         mtimes[path] = timestamp_to_string(mtime)
     row = {key: None for key in FIELDS}
-    row["IP_ADDR1"] = facts["ansible_default_ipv4"]["address"]
-    row["COMPUTER_NAME"] = facts["ansible_hostname"]
-    row["OPERATION_SYSTEM"] = facts["ansible_system"]
-    row["MAJOR_VERSION"] = 0
-    row["MINOR_VERSION"] = 0
-    row["AGENT_VERSION"] = "0.0.0.0"  # nosec
-    row["CIDS_ENGINE_VERSION"] = "0.0.0.0"  # nosec
-    row["CIDS_DRV_ONOFF"] = "ON"
-    row["LAST_SCAN_TIME"] = mtimes[LAST_SCAN_LOG_FILENAME]
-    row["LAST_VIRUS_TIME"] = mtimes[LAST_DETECTION_FILENAME]
-    row["PATTERNDATE"] = mtimes[CLAMAV_DB_FILENAME]
+    # "Group Name" is intentionally left blank so that it can be manually
+    # edited after the output CSV has been generated.
+    row["Group Name"] = ""
+    row["System Name"] = facts["ansible_hostname"]
+    row["Last Update Time"] = mtimes[CLAMAV_DB_FILENAME]
+    row["Last Scan Time"] = mtimes[LAST_SCAN_LOG_FILENAME]
+    # row["Last Detection Time"] = mtimes[LAST_DETECTION_FILENAME]
```

Comment:
Why is this commented out?

Reply:
I left it in there commented-out because it may be a useful output value later, but it is not currently something that we have been asked to provide at this time.

Reply:
Thoughts on adding an explanation? I just hate to see randomly commented out blocks of code.

Reply:
Agreed - please enjoy https://github.com/cisagov/clamav-report/pull/13/commits/dac07afc21308a3d8e24e91ad8198a23aaf90c67.